app/modules/ads/setup.js

```diff
@@ -78,7 +78,10 @@ export const adsSetup = {
 
     context.extend(defaultAdContext);
 
-    if (adsContext.opts.isAdTestWiki) {
+    if (adsContext.opts.isAdTestWiki && adsContext.targeting.testSrc) {
```

Comment:
Looking for volunteers who'll help me unit test it in https://github.com/Wikia/mobile-wiki/blob/3c7c72e9578a35d9e5ec0ad3e90cfead4bb6d830/tests/unit/modules/ads/ads/setup-test.js